src/vm/datatypes/strings.c

```diff
@@ -724,6 +724,87 @@ static Value wrapString(DictuVM *vm, int argCount, Value *args) {
     return OBJ_VAL(takeString(vm, temp, strlen(temp)));
 }
 
+static Value snakeToCamelCaseString(DictuVM *vm, int argCount, Value *args) {
+    if (argCount != 0) {
+        runtimeError(vm, "snakeToCamelCase() takes no arguments (%d given)", argCount);
+        return EMPTY_VAL;
+    }
+
+    ObjString *string = AS_STRING(args[0]);
+
+    if (string->length < 3) {
+        return OBJ_VAL(string);
```

Comment:
this is a missmatch in behaviour, because the later implementation returns a new string, this returns a pointer to the existing string if i am not mistaken.

Reply:
It's not the end of the world, strings are interned in the VM anyways